core/src/test/scala/org/apache/spark/DistributedSuite.scala

```diff
@@ -206,7 +206,8 @@ class DistributedSuite extends SparkFunSuite with Matchers with LocalSparkContex
     "caching on disk" -> StorageLevel.DISK_ONLY,
     "caching in memory, replicated" -> StorageLevel.MEMORY_ONLY_2,
     "caching in memory, serialized, replicated" -> StorageLevel.MEMORY_ONLY_SER_2,
-    "caching on disk, replicated" -> StorageLevel.DISK_ONLY_2,
+    "caching on disk, replicated 2" -> StorageLevel.DISK_ONLY_2,
+    "caching on disk, replicated 3" -> StorageLevel.DISK_ONLY_3,
```

Comment:
so what happen if there aren't 3 executors? do we have a test that needs updating?

Reply:
> so what happen if there aren't 3 executors?

The number of copies becomes 2 and this test case fail reasonably.

> do we have a test that needs updating?

Yes. This test suite is updated at line 41.